components/container/index.tsx

```diff
@@ -115,7 +115,7 @@ export default function Container({ iri }: IPodList): ReactElement {
   /* eslint react/jsx-props-no-spreading: 0 */
   return (
     <>
-      <PageHeader mutate={mutate} />
+      <PageHeader mutate={mutate} data={data} />
```

Comment:
A bit skeptic of passing variables way through multiple components like this, makes reusability a bit harder/cumbersome. Could perhaps the use of context be more approapriate? And perhaps call it something less ambigous than data?

Reply:
@daytonn do you have an opinion on this?

Reply:
@megoth I renamed the variable to `resourceList` - open to suggestions for a better name!
I also fixed the issue of the flyout not closing when saving, it does now.
Re: context - I decided against it since it is only one prop and only passing through 2 components. However if we are moving adding files to flyout instead of the current dialog, it might be the case that we can have a reusable component for both and a flyout context, but we can always revisit this when we do that work.